packages/location-field/src/styled.ts

```diff
@@ -46,8 +46,10 @@ export const MenuItemList = styled.ul.attrs({
   list-style: none;
   margin: 2px 0 0;
   min-width: 160px;
+  max-height: 75vh;
```

Comment:
Does this work if you increase the number of responses shown?

Reply:
I don't think that would impact it! I tried with a few different environments, and the container stayed consistent regardless of the geocoder results. Let me know if there's something specific I can test though